modules/navigation.js

```diff
@@ -0,0 +1,49 @@
+export default function navigation() {
+  const nav1 = document.querySelector('#list');
+  const nav2 = document.querySelector('#addNew');
+  const nav3 = document.querySelector('#contact');
+
+  const bookDisplaySection = document.querySelector('#bookSection');
+  const formDisplaySection = document.querySelector('#formSection');
+  const contactDisplaySection = document.querySelector('#contactSection');
+
+  nav1.addEventListener('click', () => {
+    if (bookDisplaySection.classList.contains('hidden')) {
+      bookDisplaySection.classList.remove('hidden');
+    }
+
+    if (!formDisplaySection.classList.contains('hidden')) {
+      formDisplaySection.classList.add('hidden');
+    }
+
+    if (!contactDisplaySection.classList.contains('hidden')) {
+      contactDisplaySection.classList.add('hidden');
+    }
+  });
+
+  nav2.addEventListener('click', () => {
+    if (formDisplaySection.classList.contains('hidden')) {
+      formDisplaySection.classList.remove('hidden');
+    }
+
+    if (!bookDisplaySection.classList.contains('hidden')) {
+      bookDisplaySection.classList.add('hidden');
+    }
+
+    if (!contactDisplaySection.classList.contains('hidden')) {
+      contactDisplaySection.classList.add('hidden');
+    }
+  });
+
+  nav3.addEventListener('click', () => {
+    if (contactDisplaySection.classList.contains('hidden')) {
+      contactDisplaySection.classList.remove('hidden');
+    }
+    if (!bookDisplaySection.classList.contains('hidden')) {
+      bookDisplaySection.classList.add('hidden');
+    }
+    if (!formDisplaySection.classList.contains('hidden')) {
+      formDisplaySection.classList.add('hidden');
+    }
+  });
+}
```

Comment:
- [ ] Please, omit the usage of the `function` keyword to declare a function instead use arrow function declaration.
- [ ] _[OPTIONAL]_ Instead of using too many if statements consider using a switch statement.

Reply:
Thank you for your comments. I understand that I have to use as more as possible ES6 syntax. However, in this particular case, it seems that a function would be more suited, from a practical and strict technical point of view. As you know it Arrow function is limited ( to be called, etc...). Please consider reviewing your recommendation for this part of the code.
I also share with you these documents from Mozilla:

1.  [Arrow function expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
2. [Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions)

Thank you for your time.

Reply:
Required asked change: made.